python_apps/rtsp-camera-to-pravega-probe.py

```diff
@@ -0,0 +1,341 @@
+#!/usr/bin/env python3
```

Comment:
Since you copied this file, I cannot easily see the changes. Rename this file back to rtsp-camera-to-pravega.py.

Reply:
done